src/components/Camera/index.js

```diff
@@ -53,9 +54,10 @@ const CameraPure = ({
   buttonType,
   onButtonClick,
   isButtonDisabled,
-  hasGrantedPermission
+  hasGrantedPermission,
+  displayBlock
 }: Props) => (
-  <div className={classNames(style.camera, className)}>
+  <div className={classNames(style.camera, { [style.displayBlock]: displayBlock }, className)}>
```

Comment:
Can't it be by default on `.camera` class so prop would not be needed?

Reply:
Good point. I'll double check on a few screen sizes, devices to be sure but I think you are right.